tools/config.py

```diff
@@ -20,9 +20,9 @@ def read_file(path):
     Read a given configuration file.
     """
     global _config
-    _config = configparser.ConfigParser()
     try:
-        _config.read(path)
+        _config = configparser.ConfigParser()
+        _config.read_file(path)
```

Comment:
`read` works just as well, because (from https://docs.python.org/3.6/library/configparser.html#configparser.ConfigParser.read):
```
If filenames is a string or [path-like object](https://docs.python.org/3.6/glossary.html#term-path-like-object), it is treated as a single filename.
```

but `read_file` is clearer, so :+1:

Reply:
Actually, `read_file` crashes the event handler when reading `app.cfg`.